indexer/services/vulcan/src/handlers/order-replace-handler.ts

```diff
@@ -70,6 +70,14 @@ export class OrderReplaceHandler extends Handler {
 
     /* Remove old order */
     const removeOrderResult: RemoveOrderResult = await this.removeOldOrder(oldOrderId);
+    if (!removeOrderResult.removed) {
+      logger.info({
```

Comment:
error log?

Reply:
I will add a comment, but i want the replacement to still succeed if this happens but wanted to log this and track a metric